CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## Python Version 1.1.6 (2023-18-11)
+
+## NPM Version 1.1.1 (2023-18-11)
+
+### Features
+
+- **python-sdk:** Created model parser extension for Google’s Gemini ([#478](https://github.com/lastmile-ai/aiconfig/pull/478), [cookbook](https://github.com/lastmile-ai/aiconfig/pull/483))
+- Added attachment field to PromptInput schema ([#473](https://github.com/lastmile-ai/aiconfig/pull/473))
+- **python-sdk:** Created batch execution interface ([#469](https://github.com/lastmile-ai/aiconfig/pull/469))
+- Added model parser for HuggingFace text2Image tasks ([#460](https://github.com/lastmile-ai/aiconfig/pull/460))
+- Updated evaluation metric values to be any arbitrary type & renamed fields ([#484](https://github.com/lastmile-ai/aiconfig/pull/484), [#437](https://github.com/lastmile-ai/aiconfig/pull/437))
+- Merged `aiconfig-extension-hugging-face-transformers` into `aiconfig-extension-hugging-face` ([#498](https://github.com/lastmile-ai/aiconfig/pull/498), [README](https://github.com/lastmile-ai/aiconfig/blob/main/extensions/HuggingFace/README.md))
+
+### Bug Fixes
+
+- Fixed caching issue by clearing outputs from prompt after `config.run()` ([#491](https://github.com/lastmile-ai/aiconfig/pull/491))
```

Comment:
This is non descriptive. A better version would be:
"Re-running the same prompt multiple times caused nondeterministic behavior due to caching outputs between runs. This has been fixed in ..."